### DISCUSSION TITLE
v3.2.0 - bunny ears

### DISCUSSION THREAD

**Discussion by VoxelPrismatic** [VoxelPrismatic/rabbit.nvim#8] [General]
# ⚞NEW⚟
- Harpoon plugin
  - Like [ThePrimeagen/Harpoon](https://github.com/theprimeagen/harpoon/tree/harpoon2), but Rabbit
- You can now set float to center
  - `rabbit.opts.window.float = "center"`
  - Thanks to @ccidral for #7

# Updates
- The History Rabbit will now recall the history of the last closed window
  - Only shown on new windows with no history
  - Option to restore full history if you closed the window accidentally
- The Reopen Rabbit will now recall the list of open buffers you last closed with
  - Separated by current working directory
  - Only available if you haven't closed a buffer yet
  - Option to open all buffers to quickly resume work
- The Oxide Rabbit is more efficient
  - **Note:** Previous history will be purged
  - Automatically deletes files that don't exist
  - Doesn't redraw the entire window on `file_del`
  - Entries don't jump around as much
- The Rabbit window now automatically removes files that don't exist
- Plugins can now have their own WinClosed autocmd

# Bug Fixes
- No longer crashes when you set `rabbit.opts.window.width` or `height` to `nil`
  - Thanks to @jeromepin for #6
- The global WinClosed autocmd now actually works
  - For how type-unsafe Lua is, I'm amazed that strings don't equal numbers

<hr /><em>This discussion was created from the release <a href='https://github.com/VoxelPrismatic/rabbit.nvim/releases/tag/v3.2.0'>v3.2.0 - bunny ears</a>.</em>

**Comment by ccidral** [comment]
> You can now set float to center

Wow, that was fast! Thank you for that ❤️ 🍺
